assets/json/03656937-19FD-4C83-9066-C76631D445EA.json

```diff
@@ -0,0 +1 @@
+{"active":true,"deny_anonymous_ids":[],"deny_quarkus_versions":[],"refresh_interval":43200.000000000}
```

Comment:
should this not be enabled true?

Reply:
also, wouldn't it make more sense to have this under `quarkus.io/uages/xx-yyy.json` ?

Reply:
this is remote config for us... We don't instruct users to change that.